GitUI/CommandsDialogs/BrowseDialog/DashboardControl/UserRepositoriesList.cs

```diff
@@ -644,5 +652,21 @@ private void tsmiRemoveFromList_Click(object sender, EventArgs e)
                 ShowRecentRepositories();
             });
         }
+
+        private void tsmiRemoveMissingReposFromList_Click(object sender, EventArgs e)
+        {
+            RepositoryContextAction(sender as ToolStripMenuItem, repository =>
+            {
+                ThreadHelper.JoinableTaskFactory.Run(async () =>
+                {
+                    foreach (Repository repo in _missingRepositories)
```

Comment:
This is not correct - missing repositories are calculated as a union of recent and favourite repos.
`RemoveRecentAsync` removes only recent repositories.